lib/services/web-view-service.ts

```diff
@@ -0,0 +1,71 @@
+///<reference path="../.d.ts"/>
+"use strict";
+import semver = require("semver");
+
+export class WebViewService implements IWebViewService {
+	
+	constructor(private $errors: IErrors,
+		private $pluginsService: IPluginsService,
+		private $project: Project.IProject,
+		private $projectConstants: Project.IProjectConstants,
+		private $options: IOptions) { }
+	
+	public get supportedWebViews(): IDictionary<IWebView[]> { 
+		return {
+			'ios': [ 
+				{ name: "Built-in", minSupportedVersion: "3.0.0", default: true }, 
```

Comment:
"Built-in" is surprisingly hard to type on the console. Can we think of something else, for example, "default"? Not a merge stopper though.